Add Docker Compose template for SPARQL endpoint and RDF browser. (#97)
w3c/EasierRDF: docker/README.md

> Simple template for a Docker Compose setup with a SPARQL Endpoint and RDF browser for a small knowledge base.

Reviewer: * probably worth saying it's Virtuoso-based
* no need for "small" qualifier, as the Virtuoso Open Source engine in this template can handle (at least) 60 Billion triples (see https://sparql.uniprot.org/ for a current deployment)
```suggestion
Simple template for a Docker Compose setup based on [Virtuoso](https://github.com/openlink/virtuoso-opensource/), providing an RDF store, SPARQL Endpoint, and RDF browser (among other available features) for a quick knowledge base deployment.
```

Author: The "small" qualifier was added because the RickView RDF browser uses a separate in-memory graph from the Sophia Rust RDF library. But because I will switch the RDF browser for another one, the qualifier can be removed.

# Run

1. `docker compose up --build`
2. test the SPARQL endpoint http://localhost:8890/sparql and the RDF browser at http://localhost:8080/

# Adapt

> 1. replace rdf/example.ttl with your knowledge base
> 2. add your namespaces to initdb.d/setup.sql
> 3. adapt docker-compose.yml
> 4. copy .env.dist to .env and change DBA\_PASSWORD

Reviewer: ```suggestion
1. replace `rdf/example.ttl` with your knowledge base
2. add your namespaces to `initdb.d/setup.sql`
3. adapt `docker-compose.yml`
4. copy `.env.dist` to `.env` and change `DBA_PASSWORD`
```

> # Run
> 
> 1. `docker compose up --build`
> 2. test the SPARQL endpoint http://localhost:8890/sparql and the RDF browser at http://localhost:8080/
> 
> # Adapt
> 
> 1. replace rdf/example.ttl with your knowledge base
> 2. add your namespaces to initdb.d/setup.sql
> 3. adapt docker-compose.yml
> 4. copy .env.dist to .env and change DBA\_PASSWORD

Reviewer: ```suggestion
# Localize

1. Replace `rdf/example.ttl` with your knowledge base
2. Add your namespaces to `initdb.d/setup.sql`
3. Adapt `docker-compose.yml`
4. Copy `.env.dist` to `.env` and change `DBA_PASSWORD`

# Start to use

1. Run `docker compose up --build`
2. Test the SPARQL endpoint at [`http://localhost:8890/sparql`](http://localhost:8890/sparql)
2. Test the RDF browser at [`http://localhost:8080/`](http://localhost:8080/)
```
